### PR TITLE
Episode 15

### PR DESCRIPTION
**Pull request by sockeqwe**
Feed seems to be valid: 
http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode15/feed.rss

TODO:

- [x] Upload mp3 file -->  @artem-zinnatullin 
- [ ] Add git tag
- [ ] Upload mp3 file in github release section

### CODE REVIEW

File: show_notes/Episode_15.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/83)
+
+In this episode Artur and Hannes do a retrospective on 2017. They share there personal highlights, favorite talks, disapointments, learnings and much more.
+
+#### Hosts:
+
+  - Artur Dryomov [@artem_zin](https://twitter.com/arturdryomov), [personal blog](http://arturdryomov.online/), [GitHub](https://github.com/ming13)
```
Comment (ming13) [inline show_notes/Episode_15.md]: Twitter handle is off.

File: show_notes/Episode_15.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Episode 15: 2017 in Retrospective
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/83)
+
+In this episode Artur and Hannes do a retrospective on 2017. They share there personal highlights, favorite talks, disapointments, learnings and much more.
```
Comment (ming13) [inline show_notes/Episode_15.md]: their personal highlights

File: show_notes/Episode_15.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  - [Two Stones, One Bird: Implementation Tradeoffs by Christina Lee - KotlinConf 2017](https://www.youtube.com/watch?v=YxOTU9F_YX4)
+  - [Testing Kotlin at Scale: Spek by Artem Zinnatullin - KotlinConf 2017](https://www.youtube.com/watch?v=R425cc6XrvA)
+  - [The Reactive Workflow Pattern](https://www.youtube.com/watch?v=KjoMnsc2lPo) (Droidcon New York 2017) and [Update](https://www.youtube.com/watch?v=mvBVkU2mCF4) (Droidcon San Francisco) by Ray Ryan 
+  - [The Resurgence of SQL by Jake Wharton and Alec Strong - Droidcon New York City 2017](https://github.com/JetBrains/kotlinconf-app)
+  - [Embracing SQL Without Abstraction by Alec Strong - Droidcon New York City 2016](https://www.youtube.com/watch?v=ksgmm8VolT4&t=840s)
+  - [OpenYOLO: Authentication made easy and secure by Iain McGinniss - Droidcon London 2017](http://uk.droidcon.com/skillscasts/10784-openyolo-authentication-made-easy-and-secure) and corresponding [blog post](https://blog.agilebits.com/2017/10/26/integrate-1password-into-your-android-apps/) 
+
+Libraries:
+
+  - [Spek](http://spekframework.org/)
```
Comment (ming13) [inline show_notes/Episode_15.md]: + Reagent?

**Comment by sockeqwe** [comment]
Thanks for your feedback. Should be addressed now

File: show_notes/Episode_15.md (hunk 3 above)
Comment (artem-zinnatullin) [reply show_notes/Episode_15.md]: wut?

**Pull request merged**